index.ts

```diff
@@ -62,13 +60,13 @@ export async function run() {
 // Fields can also contain subfields instead of pointing to a value directly.
 // This program contains a `status` field that resolves to a string.
 // Try changing the string returned, hit save, and see your text in the Explorer next to `getting-started`.
-// `status` is a default field that all Membrane programs have, but you can add any fields to a schema.
 
 export function status() {
   return "👋 start here!";
 }
 
-// To learn more about actions in Membrane, visit: https://www.membrane.io/the-graph
+// Note: `status` is a default field that all Membrane programs have, but you can add any fields to a schema.
+// To learn more about actions and fields in Membrane, visit: https://docs.membrane.io/concepts/schema
```

Comment:
Normally I use the word "implicit" when talking about `status`, `endpoint` and `email`. I think it better conveys how they work.